bgpd/bgp_mplsvpn.c

```diff
@@ -1616,14 +1618,40 @@ void vrf_import_from_vrf(struct bgp *to_bgp, struct bgp *from_bgp,
 	 */
 	vname = (from_bgp->name ? XSTRDUP(MTYPE_TMP, from_bgp->name)
 			       : XSTRDUP(MTYPE_TMP, VRF_DEFAULT_NAME));
+	if (!listcount(to_bgp->vpn_policy[afi].import_vrf))
+		listnode_add(to_bgp->vpn_policy[afi].import_vrf, vname);
+	else {
+		is_inst_match = false;
+		for (ALL_LIST_ELEMENTS_RO(to_bgp->vpn_policy[afi].import_vrf,
```

Comment:
How is this possible?  in the only place we call this function we already do this search.  is_inst_match *can* never be true here.  Look at bgp_imexport_vrf_cmd in bgp_vty.c